### ISSUE TITLE
Long message handler error. I think "j.json()" should be r.json()

### ISSUE THREAD

**Issue by MikeLeahy24** [dwreeves/blsconnect#1]
File "C:\Users\Michael Leahy\AppData\Local\Programs\Python\Python39\lib\site-packages\blsconnect\request.py", line 127, in series
    r[s_y] = self._request(series, s_y, e_y, catalog)
  File "C:\Users\Michael Leahy\AppData\Local\Programs\Python\Python39\lib\site-packages\blsconnect\request.py", line 213, in _request
    print(j.json()['message'][0])
NameError: name 'j' is not defined

**Comment by dwreeves** [comment]
yeah I don't maintain this anymore. I built this long before I understood good software practices that would have avoided such a thing. One day maybe that gets fixed, but unlikely.
